### PR TITLE
feat: strf-9087 stencil localhost only shows default storefront data

### PR DESCRIPTION
**Pull request by jairo-bc**
#### What?

Added ability to specify channel id by running `stencil start --chanelId [value]`
If not specified and enabled MSF option, cli will prompt to select one.

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

-   [https://jira.bigcommerce.com/browse/STRF-9087](STRF-9087)

#### Screenshots (if appropriate)

Screenshot 2021-06-04 at 17 00 08
Screenshot 2021-06-04 at 17 00 17
Screenshot 2021-06-04 at 17 00 51

### CODE REVIEW

File: lib/stencil-push.utils.js [hunk 1]
```diff
@@ -339,8 +339,13 @@ utils.promptUserForChannel = async (options) => {
         return options;
     }
 
+    const selectedChannelId = await utils.promprtUserToSelectChannel(channels);
+    return { ...options, channelId: selectedChannelId };
+};
+
+utils.promprtUserToSelectChannel = async (channels) => {
```
Comment (bookernath) [inline lib/stencil-push.utils.js:346]: typo in method name, should be `promptUserToSelectChannel`

**Comment by jairo-bc** [comment]
Screenshot 2021-06-07 at 18 22 51
Screenshot 2021-06-07 at 18 25 00

File: server/plugins/renderer/renderer.module.js [hunk 2]
```diff
@@ -385,6 +385,8 @@ internals.buildReqHeaders = ({
         headers['stencil-config'] = JSON.stringify(stencilConfig);
     }
 
+    console.log();
```
Comment (junedkazi) [inline server/plugins/renderer/renderer.module.js:388]: We need to remove this log

File: server/plugins/renderer/renderer.module.spec.js [hunk 3]
```diff
@@ -219,6 +245,8 @@ describe('Renderer Plugin', () => {
         });
 
         it('should send a request to the storefront server with correct headers', async () => {
+            console.log('hherer');
+            console.log('hherer');
```
Comment (junedkazi) [inline server/plugins/renderer/renderer.module.spec.js:249]: remove these console logs

**Review by junedkazi — APPROVED** [review]

**Review by bookernath — APPROVED** [review]
LGTM, please squash commits on merge.

**Pull request merged**